hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -834,10 +844,22 @@ if (withReplica) {
 </tbody>
 </table>
 <% }
-} catch(Exception ex) {
+} catch(Exception ex) { %>
+  Unknown Regions
```

Comment:
This is basically some issue with regions while iterating regDistribution. Is it good to display: `Unknown issue with Regions` instead?

Reply:
Thanks for your comment. 
I'll change the message to "Unknown issue with Regions"